ksiren/build.gradle

```diff
@@ -8,14 +8,19 @@ def description = 'Main package of KSiren: A Kotlin library for parsing response
 def versionValue = "2.0.0"
 
 dependencies {
-    compile 'org.jetbrains.kotlin:kotlin-stdlib:1.3.31'
+	compile 'org.jetbrains.kotlin:kotlin-stdlib:1.3.31'
+	compile 'org.apache.commons:commons-text:1.9'
```

Comment:
I'd rather not add a relatively heavy dependency for a minimal amount of functionality, but lighter-weight alternatives (eg. `org.json`) seem to have dependency conflicts in consumers, and I'd rather depend on a well-tested library for this. That said, I'll look at other lighter alternatives.

Ideally, I would think the GSON/Moshi adapters could handle automatic model-based serialization, and/or `JSONObject` could be used to some effect, but those are larger short-term lifts based on the current deserialization patterns.